src/test/java/com/stripe/model/LegalEntityTest.java

```diff
@@ -0,0 +1,69 @@
+package com.stripe;
+
+import com.stripe.exception.StripeException;
+import com.stripe.model.Account;
+import com.stripe.model.LegalEntity;
+import com.stripe.net.APIResource;
+import com.stripe.net.RequestOptions;
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.*;
+
+public class LegalEntityTest extends BaseStripeTest {
+	static String legalEntityHash =
+		"{" +
+		"	\"type\": \"sole_prop\"," +
+		"	\"business_name\": \"business name\"," +
+		"	\"address\": {" +
+		"		\"line1\": \"12 Grove Street\"," +
+		"		\"line2\": null," +
+		"		\"city\": \"New York\"," +
+		"		\"state\": \"NY\"," +
+		"		\"postal_code\": \"10014\"," +
+		"		\"country\": \"US\"" +
+		"	}," +
+		"	\"first_name\": \"Monica\"," +
+		"	\"last_name\": \"Geller\"," +
+		"	\"dob\": {" +
+		"		\"day\": 4," +
+		"		\"month\": 4," +
+		"		\"year\": 1969" +
+		"	}," +
+		"	\"additional_owners\": [" +
+		"		]," +
+		"	\"verification\": {" +
+		"		\"status\": \"verified\"," +
+		"		\"document\": null," +
+		"		\"details\": null" +
+		"	}" +
+		"}";
+
+	@Test
+	public void testDeserialize() throws StripeException {
+		LegalEntity le = APIResource.GSON.fromJson(legalEntityHash, LegalEntity.class);
+
```

Comment:
Would it be better to just construct an expected version of the object, implement `.equals`, then `assertEquals`?